1-org/org_policy.tf

```diff
@@ -149,9 +149,13 @@ module "org_enforce_bucket_level_access" {
   Access Context Manager Policy
 *******************************************/
 
+resource "random_id" "suffix" {
+  byte_length = 4
+}
+
 module "access_context_manager_policy" {
   source      = "terraform-google-modules/vpc-service-controls/google"
   version     = "~> 2.0.0"
-  parent_id   = local.organization_id
-  policy_name = "default policy"
+  parent_id   = var.org_id
+  policy_name = "default policy-${random_id.suffix.hex}"
```

Comment:
nit: the random suffice is not really required for this resource given that only one can exist in an org

Reply:
removed